roboteam_ai/src/coach/heuristics/OffensiveScore.cpp

```diff
@@ -3,6 +3,7 @@
 //
 
 #include "OffensiveScore.h"
+#include "../OffensiveCoach.h"
```

Comment:
I dislike that we need the coach object here since the idea is to make this class just calculate scores. This creates a circular dependency.